BrainPortal/spec/controllers/users_controller_spec.rb

```diff
@@ -586,7 +588,7 @@
         end
 
         it "should not allow site manager to destroy a user not from the site" do
-          expect { delete :destroy, params: {:id => user.id }}.to raise_error(ActiveRecord::RecordNotFound)
+          expect { delete :destroy, params: {:id => user.id }}
```

Comment:
This is an error: this expect() does not test anything, the test will never fail.

Reply:
fixing

Reply:
fixing